profiles/utilities.py

```diff
@@ -1,2 +1,11 @@
+from typing import Type
+
+
+def chain_exception(exception: Type[Exception], previous: Exception, message: str = None):
+    exception = exception(message or str(previous))
+    exception.__cause__ = previous
```

Comment:
See also #11. `from` only works with `raise`.

Reply:
I take this in the sense that it cannot be used to create an exception object outside of the `raise` construct, where it does as syntactic sugar what is being done here

Reply:
Yep. This is a little redundant now as we don't actually do anything with the previous exception, but when we add logging etc it will be useful.